### PR TITLE
Generate and update kuberhealthy crds

### PR DESCRIPTION
**Pull request by joshulyne**
Addresses #778 & #351

### CODE REVIEW

File: cmd/kuberhealthy/Makefile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 IMAGE="kuberhealthy/kuberhealthy"
-TAG="unstable"
+TAG="crds-update"
```
Comment (jonnydawg) [inline cmd/kuberhealthy/Makefile:2]: maybe change it back to unstable

File: pkg/apis/khjob/v1/doc.go [hunk 2]
```diff
@@ -1,5 +1,5 @@
 // +k8s:deepcopy-gen=package
 // +k8s:defaulter-gen=TypeMeta
-// +groupName=khjob
+// +groupName=comcast.github.io
```
Comment (jonnydawg) [inline pkg/apis/khjob/v1/doc.go:3]: did we still want our apigroup to be `comcast.github.io` or do we want to change it to `kuberhealthy.io`

Comment (integrii) [reply pkg/apis/khjob/v1/doc.go:3]: this is a breaking change so we'll have to save it for v3...

File: pkg/checks/external/util/main.go [hunk 3]
```diff
@@ -110,7 +110,7 @@ func PodNameExists(client *kubernetes.Clientset, podName string, namespace strin
 	}
 
 	if p.Name == "" {
-		log.Warningln("Pod name is empty. Pod does not exist")
+		log.Warnln("Pod name is empty. Pod does not exist")
```
Comment (integrii) [inline pkg/checks/external/util/main.go:113]: potentially drop the warning here and the one above on line 108?

File: pkg/health/state.go [hunk 4]
```diff
@@ -16,15 +16,17 @@ import (
 	"net/http"
 
 	log "github.com/sirupsen/logrus"
+
+	khstatev1 "github.com/Comcast/kuberhealthy/v2/pkg/apis/khstate/v1"
```
Comment (integrii) [inline pkg/health/state.go:20]: Remove `Comcast` here for `kuberhealthy`

**Review by integrii — CHANGES_REQUESTED** [review]

**Comment by joshulyne** [comment]
use camelcase for khstate struct.

**Review by integrii — APPROVED** [review]
This is a big one!  Thanks! 🚀

**Pull request merged**